ddp-workspace/projects/ddp-pancan/src/app/components/welcome/welcome.component.scss

```diff
@@ -0,0 +1,391 @@
+@import '../../../styles/variables.scss';
+
+$container-width: 1200px;
+
+.main > * {
+    width: 100%;
+}
+
+section {
+    box-sizing: border-box;
+    text-align: center;
+
+    &:nth-of-type(even) {
+        background-color: #F6F7FA;
+    }
+
+    &.introduction {
+        max-width: 1440px;
+        padding-top: 108px;
+        padding-bottom: 18px;
+        text-align: center;
+        background-image: url(/assets/images/hero.png);
+        background-repeat: no-repeat;
+        background-position: bottom;
+        background-size: contain;
+
+        .title {
+            margin-top: 0;
+            margin-bottom: 16px;
+        }
+
+        .logos {
+            img:first-child {
+                margin-right: 5rem;
+            }
+        }
+    }
+
+    &.learn-more {
+        padding: 80px 0 65px;
+
+        .title {
+            margin-bottom: 32px;
+        }
+    }
+
+    &.join {
+        height: 470px;
+        padding: 36px 40px 55px 100px;
+        display: flex;
+        box-sizing: border-box;
+
+        .image {
+            flex: 0 1 400px;
+            margin-right: 100px;
+            background-image: url(/assets/images/map.png);
+            background-repeat: no-repeat;
+            background-position: top;
+            background-size: contain;
+        }
+
+        .description {
+            flex: 1 2 auto;
+            padding-top: 70px;
+        }
+
+        .title {
+            margin-bottom: 34px;
+        }
+        .text {
+            text-align: left;
+
+            &:first-of-type {
+                margin-bottom: 32px;
+            }
+        }
+    }
+
+    &.participate {
+        padding: 112px 20px;
+        text-align: left;
+
+        .title {
+            margin-bottom: 78px;
+        }
+
+        .container {
+            max-width: $container-width;
+            margin: 0 auto;
+            color: #000000;
```

Comment:
I'd recommend setting some default color to the body element, together with font-family, check how other studies implement it

Reply:
Good point. Fixed at ee510c5723c7990f01eb018fab32b3622424d9a9